packages/grafana-data/src/datetime/moment_wrapper.ts

```diff
@@ -96,7 +97,7 @@ export const toUtc = (input?: DateTimeInput, formatInput?: FormatInput): DateTim
 };
 
 export const toDuration = (input?: DurationInput, unit?: DurationUnit): DateTimeDuration => {
-  return moment.duration(input as DurationInputArg1, unit) as DateTimeDuration;
+  return moment.duration(input as DurationInputArg1, unit as DurationInputArg2) as DateTimeDuration;
```

Comment:
seems like moment types don't include `isoWeek`, but it's actually present in the source, that's also what we were using before strongly typing https://github.com/grafana/grafana/pull/31709/files#diff-56a4a2354a8d6139680ca2c9727bec82efb5ffd660ee1dc66b5508bbe8c0ce06R12 not sure we have a better way of fixing this.

Reply:
Would put this as a comment to explain the type cast here.

Reply:
good point, thanks!